samples/dashboards/sync/custom-component-sync/demo.js

```diff
@@ -0,0 +1,369 @@
+                    return `
+                        <b>${point.name}</b><br>
+                        Population: <b>${formatBigNumber(point.x)}</b><br>
+                        Happiness: <b>${point.y}</b>
+                    `;
+                }
+            }
+        },
+        sync: {
+            customSync: {
+                enabled: true,
+                syncedColumn: 'Happiness',
+                handler: function () {
+                    const { board } = this;
+                    const { dataCursor: cursor } = board;
+                    const table = this.connector && this.connector.table;
+                    const chart = this.chart;
+                    const syncedColumnName =
+                        this.sync.syncConfig.customSync.syncedColumn;
+
+                    if (
+                        !table || !board || !chart ||
+                        !this.sync.syncConfig.customSync.enabled
+                    ) {
+                        return;
+                    }
+
+                    let xValue,
+                        leftAverage,
+                        rightAverage;
+
+                    const drawLines = () => {
+                        const xAxis = chart.xAxis[0];
+                        const yAxis = chart.yAxis[0];
+                        const bandXPos = xAxis.toPixels(xValue);
+                        const leftYPos = yAxis.toPixels(leftAverage);
+                        const rightYPos = yAxis.toPixels(rightAverage);
+
+                        const mirrorBandD = [
+                            'M', xAxis.toPixels(xValue), xAxis.top,
+                            'l', 0, xAxis.height
+                        ];
+
+                        const leftBandD = [
+                            'M', xAxis.left, leftYPos,
+                            'L', bandXPos, leftYPos
+                        ];
+
+                        const rightBandD = [
+                            'M', bandXPos, rightYPos,
+                            'L', xAxis.width + xAxis.left, rightYPos
+                        ];
+
+                        if (!xAxis.mirrorBand) {
+                            xAxis.mirrorBand = chart.renderer.path().attr({
+                                stroke: '#f25',
+                                zIndex: 3,
+                                'stroke-dasharray': 5
+                            }).add();
+
+                            xAxis.leftBand = chart.renderer.path().attr({
+                                stroke: '#f25',
+                                zIndex: 3,
+                                'stroke-width': 2
+                            }).add();
+
+                            xAxis.rigthBand = chart.renderer.path().attr({
+                                stroke: '#f25',
+                                zIndex: 3,
+                                'stroke-width': 2
+                            }).add();
+
+                            xAxis.leftLabel = chart.renderer.text().attr({
```

Comment:
Swap alignment on these two labels. Should be positioned next to the vertical band.

Also make them slightly larger and add a white outline so that text is readable when placed over scatter dots.

Reply:
I would prefer a white outline here. Looks cleaner and at the same time adds some space between digits. Easier to read.

Reply:
What about dark mode? Currently it's white only in dark mode, according to the contrast to the background. Should I set it to white for both modes?

Reply:
The outline should not contrast with background, but blend into it. Will make it easier to read when positioned over dots. If not over dots you will not see the outline. So, it should be white in lightmode and dark in darkmode.